build.sh

```diff
@@ -155,6 +155,13 @@ if [[ $BUILD_TARGET == 'hera' ]]; then
   ln -sf $RDASAPP_TESTDATA/crtm $dir_root/bundle/test-data-release/crtm
 fi
 
+# Set lower number of build jobs on Orion due to memory limit on login nodes
+if [[ $BUILD_TARGET == 'orion' ]]; then
+  BUILD_JOBS=4
+else # hera, hercules, jet
+  BUILD_JOBS=6
+fi
+
```

Comment:
@SamuelDegelia-NOAA I think we will not need line 158-163 if we add an option "-j" and then we can keep line 171 and line 175.
If this is urgent, we may add "-j" first and update build_and_test.md

Reply:
you can also do, without even a PR:
`BUILD_JOBS=4 ./build.sh`

Reply:
@CoryMartin-NOAA  Yes, that works. 
It is rare, but there are still some guys using `csh`, :)
So `build.sh -j4` may be simpler. Just my 2 cents.

Reply:
I can add the `-jN` option in a future PR. Also, in my opinion I like keeping the default option in the script for Orion since that machine will (likely) always need the lower value. We could add info to the readme to do something like `BUILD_JOBS=4 ./build.sh` but I think someone could miss that, and keeping it as simple as just running `build.sh` would be preferable.